site/content/integrate/apps/deployment/_index.md

```diff
@@ -49,40 +49,71 @@ For a go app the manifest snippet would look like this:
         }
     ]
 }
-``` 
+```
 
-### Deploying third-party apps to AWS
+### Deploying third-party apps to third-party AWS cloud environment
 
-Note that third-party apps are not supported in the Mattermost Cloud - they're used only for developer testing convenience. Provisioning in the third-party AWS cloud environment is done by the **appsctl** tool using the command:
+Note that third-party apps are not supported in the Mattermost Cloud - they're used only for developer testing convenience. Provisioning in the third-party AWS cloud environment is done by the **appsctl** tool after completing the environment set up.
 
-```bash
-go install github.com/mattermost/mattermost-plugin-apps/cmd/appsctl@latest
-appsctl provision app /PATH/TO/YOUR/APP/BUNDLE
-```
+We need an app bundle to provision an app. The bundle might be provisioned from the local disk, from S3 (not implemented yet), or from a URL (not implemented yet). Provisioning consists of three parts which will be completed via the **appsctl** tool:
+
+1. Creating the lambda functions with appropriate policies.
+2. Storing static assets in the dedicated S3 bucket.
+3. Storing the app’s manifest file in the same dedicated S3 bucket.
+
+#### Mattermost set up
+
+Follow the Prerequisites section of the [JS](../quick-start-js) or [GO](../quick-start-go) Quick Start Guides to set up your Mattermost instance.
```

Comment:
One must also install the apps plugin: https://developers.mattermost.com/integrate/apps/quick-start-js/#install-the-apps-plugin

Reply:
Added by Justine in [`cc17586` (#826)](https://github.com/mattermost/mattermost-developer-documentation/pull/826/commits/cc17586fb1b4135c8cd630ea339cd3e195892763)